src/Microsoft.DotNet.Build.Tasks/PackageFiles/packageresolve.targets

```diff
@@ -2,6 +2,8 @@
 <Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
   <UsingTask TaskName="PrereleaseResolveNuGetPackageAssets" AssemblyFile="$(ToolsDir)Microsoft.DotNet.Build.Tasks.dll"/>
+  <UsingTask TaskName="ValidateProjectDependencyVersions" AssemblyFile="$(ToolsDir)Microsoft.DotNet.Build.Tasks.dll"/>
```

Comment:
You will need to update this to BuildToolsTaskDir after @joperezr's https://github.com/dotnet/buildtools/pull/342 change.